app/domain/authentication/authn_k8s/execute_command_in_container.rb

```diff
@@ -11,10 +11,13 @@
 module Authentication
   module AuthnK8s
 
-    KubeExec ||= CommandClass.new(
+    ExecuteCommandInContainer ||= CommandClass.new(
```

Comment:
Side note that is not part of this PR:
```rb
extend Forwardable
def_delegators :@k8s_object_lookup, :kube_client
```
We should remove this, since it all add another public method to the `CommandClass`, which is supposed to have only a `call` method.  Instead, we should either define `kube_client` privately, or simply use the long form `@k8s_object_lookup.kube_client` (there are only 2 references to `kube_client` so this is fine).

Reply:
Another existing code comment:

1. We should be able to remove all arguments from `kube.on_open(ws_client, body, stdin)`, because the kube instance already has access to all of them.
2. We should be able to remove `ws_client` arg from `kube.on_message(msg, ws_client)` for the same reason.